### PR TITLE
Update Darts lib version `0.22.0` --> `0.23.0`

### PR DESCRIPTION
**Pull request by martinb-bb**
Updating version to support new features and updates. 

Should not affect the installer...

### CODE REVIEW

**Comment by jmaslek** [comment]
Anaconda tests are failing because it can't install finviz.  Did you locally pin that one by any chance?

**Comment by martinb-bb** [comment]
> Anaconda tests are failing because it can't install finviz. Did you locally pin that one by any chance?

Nope. never touched finviz! Rebuilding my env locally in conda and installing all works.

File: requirements.txt [hunk 1]
```diff
@@ -1,266 +1,259 @@
...
-executing==1.1.1; python_version >= "3.8"
+exchange-calendars==4.2.4; python_version >= "3.8" and python_version < "4.0" and python_full_version >= "3.7.0"
+executing==1.2.0; python_version >= "3.8"
 fastjsonschema==2.16.2; python_full_version >= "3.7.0" and python_version >= "3.7"
 feedparser==6.0.10; python_version >= "3.6"
 ffn==0.3.6; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.6.0"
 financedatabase==1.0.2
 finnhub-python==2.4.15
-finviz==1.4.4
+finviz==1.4.5; python_version >= "3.8" and python_version < "4.0"
```
Comment (jmaslek) [inline requirements.txt:60]: liar :P

Comment (martinb-bb) [reply requirements.txt:60]: That's all poetry! I swearrrrrr

Comment (jmaslek) [reply requirements.txt:60]: been there.  

what poetry functions you run?

Comment (martinb-bb) [reply requirements.txt:60]: Just changed tml version to darts=0.23.0

Then poetry update. 

Anything else i should have done?

Comment (jmaslek) [reply requirements.txt:60]: So I broke this recently by doing `poetry update`. I think you just want to do poetry update on darts, or else it resolves the entire chain and updates everything.  So just `poetry update darts` (or add the right version of darts to poetry)

**Comment by jmaslek** [comment]
Since you are updating deps, can you also do this one:

#3856

**Comment by martinb-bb** [comment]
> Since you are updating deps, can you also do this one:
> 
> #3856

yup! ill make a new PR and close this one. start fresh and do it right this time.

**Pull request closed without merging**